CMakeLists.txt

```diff
@@ -7,6 +7,14 @@ ELSE(TAOS_CI_INSTALL_DIR)
     SET(TAOS_CI_INSTALL_DIR "/var/www/html/TAOS-CI")
 ENDIF(TAOS_CI_INSTALL_DIR)
 
+IF (NOT CMAKE_INSTALL_SYSCONFDIR)
```

Comment:
need to keep indentation. exsting 'if' statement doesn't have blank. 
ex) IF(NOT CMAKE_INSTALL_SYSCONFDIR)

Reply:
Nice catch.